infra/cardano-sl-infra.cabal

```diff
@@ -126,6 +126,8 @@ library
                       , formatting
                       , generic-arbitrary
                       , hashable
+                      , http-client
```

Comment:
We depend from `http-client` and `http-client-tls` anyway, so this doesn't add any extra dep, in fact.